test/test-e2e.sh

```diff
@@ -23,9 +23,10 @@ setup_helm() {
   if ! [[ $(${HELM} version |grep Version |awk -F'Version:' '{print $2}' |awk -F',' '{print $1}') == "\"v${HELM_VERSION}\"" ]] ; then
       echo "helm not found or bad version, downloading binary"
       mkdir -p _output
-      wget https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz
-      tar -zxvf helm-v${HELM_VERSION}-linux-amd64.tar.gz
-      mv linux-amd64/helm _output/helm
+      curl -Lo /tmp/helm.tar.gz "https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz"
+      tar -xzf /tmp/helm.tar.gz -C /tmp
+      rm -f /tmp/helm.tar.gz
+      mv -f /tmp/linux-amd64/helm _output/helm
       chmod +x _output/helm
       HELM=_output/helm
```

Comment:
This could probably be simplified by doing something like:

```suggestion
      wget -qO- "https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz" | tar xz -C _output/helm
      chmod +x _output/helm/linux-amd64/helm
      HELM=_output/helm/linux-amd64/helm
```

Reply:
Is there a reason for `wget` over `curl?

Reply:
We can use both, I just used the one I had in my term history. You would just have to use curl with `-s`